packages/react-components/src/Expander.tsx

```diff
@@ -78,6 +97,8 @@ function Expander ({ children, className = '', help, helpIcon, isOpen, isPadded,
         className='ui--Expander-summary'
         onClick={toggleExpanded}
       >
+        {iconPlacement === 'left' && <ExpandIcon className='ui--Expander-icon-left'
+          isExpanded={isExpanded} />}
```

Comment:
Wth multiple attributes, attribute each on a single line with () wrapping for readability.